src/cli/commands/global.js

```diff
@@ -69,44 +72,44 @@ async function getBins(config: Config): Promise<Set<string>> {
   return paths;
 }
 
-function getGlobalPrefix(config: Config, flags: Object): string {
+async function getGlobalPrefix(config: Config, flags: Object): Promise<string> {
   if (flags.prefix) {
     return flags.prefix;
   } else if (config.getOption('prefix')) {
     return String(config.getOption('prefix'));
   } else if (process.env.PREFIX) {
     return process.env.PREFIX;
-  } else if (process.platform === 'win32') {
+  }
+
+  let prefix = FALLBACK_GLOBAL_PREFIX;
+  if (process.platform === 'win32') {
+    // %LOCALAPPDATA%\Yarn --> C:\Users\Alice\AppData\Local\Yarn
     if (process.env.LOCALAPPDATA) {
-      return path.join(process.env.LOCALAPPDATA, 'Yarn', 'bin');
+      prefix = path.join(process.env.LOCALAPPDATA, 'Yarn');
```

Comment:
@BYK The standard on Windows is to use uppercase. If you take a look in `%LocalAppData%` you'll see that most folders are uppercase. Some badly-behaved apps use lowercase, but it's not the norm.